inc/azure_uamqp_c/sasl_server_mechanism.h

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#ifndef SASL_SERVER_MECHANISM_H
+#define SASL_SERVER_MECHANISM_H
+
+#ifdef __cplusplus
+extern "C" {
+#include "cstdint"
```

Comment:
Ha. Actually I did not added it on purpose. STL had a comment that we do not really need it and the header will be soon deprecated.

---
In reply to: [124315273](https://github.com/Azure/azure-uamqp-c/pull/145#discussion_r124315273) [](ancestors = 124315273)